bindings/pulp/bindings/static.py

```diff
@@ -0,0 +1,18 @@
+from pulp.bindings.base import PulpAPI
+
+
+class StaticRequest(PulpAPI):
+    """
+    Connection class to access static calls
+    """
+    def __init__(self, pulp_connection):
+        super(StaticRequest, self).__init__(pulp_connection)
```

Comment:
This method doesn't seem to do anything specific to this subclass. I recommend removing it.